src/type_sys/fender_value/fender_strings.rs

```diff
@@ -0,0 +1,247 @@
+use std::{cell::RefCell, fmt::Display, hash::Hash, ops::Index};
+
+use crate::index_oob;
+
+#[derive(Debug, Clone)]
+pub struct FenderString {
+    chars: Vec<char>,
+    cacheable: bool,
```

Comment:
So is the point that you can create some strings that will never be cached? I'm not really sure why we need this field.